python-regression/tests/features/machine6/config.yml

```diff
@@ -16,7 +16,7 @@ default_args: &args
   ]
 
 default_ixi: &ixi
-  ['https://github.com/DyrellC/LocalSnapshots.ixi/archive/v0.1.tar.gz']
+  ['https://raw.githubusercontent.com/iotaledger/iri/dev/python-regression/IXI/LocalSnapshots.ixi.tar.gz']
```

Comment:
Hmm,
I didn't really look at TIAB logic but I imagined that the IXI will be added to the repo in an untarred form, this way it could easily be read by anyone.

I'll speak with @karimodm about this and then we will decide if we need to change, or merge the PR as it is